src/sentry/api/endpoints/group_details.py

```diff
@@ -235,6 +236,17 @@ def get(self, request: Request, group) -> Response:
                 )
                 data.update({"sentryAppIssues": sentry_app_issues})
 
+            if "hasAttachments" in expand:
+                if not features.has(
+                    "organizations:event-attachments",
+                    group.project.organization,
+                    actor=request.user,
+                ):
+                    return self.respond(status=404)
+
+                attachments = EventAttachment.objects.filter(group_id=group.id)
+                data.update({"hasAttachments": len(attachments) > 0})
```

Comment:
Since this is a one way update (from 0 to 1), could we avoid the `data.update` if `len(attachments) == 0`?

Reply:
That assumes `data.update` does a DB round trip btw, not sure if it helps avoiding it

Reply:
i think if we don't have the `data.update` then we won't put the property onto the item at all - the idea is to have `hasAttachments` be a flag on the issue, and show whether it's true or false if the user has specifically requested to expand it

Reply:
@aliu3ntry yeah they can

Reply:
Makes sense! thanks folks